sw/device/lib/dif/autogen/dif_lc_ctrl_autogen.c

```diff
@@ -0,0 +1,9 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// This file is auto-generated.
```

Comment:
I wonder if it would be useful to get this into the style guide somewhere... i.e. this exact text must occur in all generated files. Or something like that. Consitency!

Reply:
Agreed, maybe I should file a separate issue on this? The format I used here follows the format used in hw/ip/prim/util/primgen/prim_pkg.sv.tpl and a few other places. But agreed that consistency throughout is best.

Reply:
The sources generated with `topgen` already have this banner. I will make a PR to sort this out.